deployments/server/templates/configmap.yaml

```diff
@@ -10,4 +10,5 @@ data:
     httpPort: {{ .Values.httpPort }}
     monitoringPort: {{ .Values.monitoringPort }}
     ollamaServerAddr: {{ .Values.ollamaServerAddr }}
+    inferenceEngineServerAddr: {{ .Values.inferenceEngineServerAddr }}
```

Comment:
This one is not set? Also is this the internal gRPC endpoint or not?

Reply:
Removed.